src/lib/browser.ts

```diff
@@ -145,6 +153,24 @@ export class Browser extends EventEmitter {
         this.emit('down', service)
     }
 
+    private filterService(service: Service) {
+        if(this.txtQuery === undefined) return
+        let serviceTxt = service.txt
+        let query = Object.entries(this.txtQuery)
+            .map(([key, value]) => {
+                try {
```

Comment:
I would avoid silently hiding an exception here and make the code more robust instead.